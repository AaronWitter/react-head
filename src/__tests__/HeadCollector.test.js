// import React from 'react';
// import HeadCollector from '../HeadCollector';

describe('HeadCollector', () => {
  it('provides context object');
  it('adds HeadTags to given array from component tree');
});
