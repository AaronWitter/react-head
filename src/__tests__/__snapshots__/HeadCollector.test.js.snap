// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeadCollector adds HeadTags to given array from component tree 1`] = `
Array [
  <tag1
    another="value1"
    name="name1"
  />,
  <tag2
    test="test2"
    third="value2"
  />,
]
`;
